src/variables.ts

```diff
@@ -15,8 +14,6 @@ export class AthenaVariableSupport extends CustomVariableSupport<DataSource, Ath
   editor = VariableQueryCodeEditor;
 
   query(request: DataQueryRequest<AthenaQuery>): Observable<DataQueryResponse> {
-    // fill query params with default data
```

Comment:
Not sure about this part - it doesn't look like it breaks anything (since this method isn't called before the default query is set it <VariableQueryEditor/>, but I might be missing a use case.

Reply:
This method is called immediately when the dashboard is loaded, so it's best to apply the default query here too.